internal/corazawaf/waf.go

```diff
@@ -420,9 +420,15 @@ func (w *WAF) SetDebugLogPath(path string) error {
 		w.Logger.SetOutput(io.Discard)
 		return nil
 	}
+
+	if path == "/dev/stdout" {
```

Comment:
This is the important change.

Reply:
What about using stderr? Should this deserve also a case for it?